keel-docker/src/main/kotlin/com/netflix/spinnaker/keel/docker/ContainerProvider.kt

```diff
@@ -29,10 +29,8 @@ data class ReferenceProvider(
   val reference: String
 ) : ContainerProvider()
 
+// used in titus handler as a way to represent a fully specified container
 @JsonDeserialize(using = JsonDeserializer.None::class)
-@Deprecated("Non-reference-based artifact providers are no longer supported.",
```

Comment:
@emjburns I took this opportunity to remove this deprecation warning based on your comments in #1310.

Reply:
Thanks